src/client/components/app-core/text.css

```diff
@@ -1,3 +1,20 @@
 html {
-  font-family: sans-serif;
+  font-family: 'system', sans-serif;
 }
+
+a {
+  font-weight: bold;
+  font-size: 1rem;
+  color: var(--black);
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+a:hover {
```

Comment:
For accessibility, always set both focus and hover so it also works for people working with a different input than just a mouse:

```css
a:focus,
a:hover {
 ...
}
```